pupa/utils/generic.py

```diff
@@ -72,3 +73,20 @@ def convert_pdf(filename, type='xml'):
     data = pipe.read()
     pipe.close()
     return data
+
+
+def canonize_url(url):
+    split_url = urlsplit(url)
+
+    canonical_query = urlencode(sorted(parse_qsl(split_url.query)))
```

Comment:
usecase: to compare two sources (as we do for people and orgs that are `source_identified`), we have to count on the ordering of params to be deterministic. `requests.response` doesn't guarantee that, though. it's actually most likely that the ordering will change.